src/components/FindWallet/WalletFilterFeature.tsx

```diff
@@ -176,7 +176,7 @@ const WalletFilterFeature = ({
                                     eventAction: `${filterOption.title}`,
                                     eventName: `Toggle ${
                                       item.title
-                                    } ${!item.showOptions}`,
+                                    } to ${item.showOptions}`,
```

Comment:
<img width="202" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/006470df-cfc3-4cdc-bc7c-38576f28f5b3">

Tested all the different options we have... this event is only fired for the `Browser` `Desktop` or `Mobile` switches shown here